lib/msal-browser/src/event/EventMessage.ts

```diff
@@ -16,7 +16,7 @@ export type EventMessage = {
     timestamp: number;
 };
 
-export type EventPayload = PopupRequest | RedirectRequest | SilentRequest | SsoSilentRequest | EndSessionRequest | AuthenticationResult | null;
+export type EventPayload = PopupRequest | RedirectRequest | SilentRequest | SsoSilentRequest | EndSessionRequest | AuthenticationResult | Window | null;
```

Comment:
Why do we emit the window object?

Reply:
I figured the `POPUP_OPENED` event was a good opportunity to address #2626 and would also allow applications to write their own logic to close the popup if it doesn't get closed for whatever reason.

Reply:
I see. Can we create a new type, e.g. `PopupEvent`, that contains the window object as a property? To future proof in case we want/need to include other info.

Reply:
Done